### PR TITLE
Fix crash on Linux when switching between tabs in printer config

### LINKED ISSUE #6261: Frequent crashes when switching tabs in preset windows
### Is there an existing issue for this problem?

- [x] I have searched the existing issues

### OrcaSlicer Version

2.1.1

### Operating System (OS)

Linux

### OS Version

Manjaro (Arch) [kernel 6.9.9] + KDE plasma [v 6.0.5]

### Additional system information

OS: Manjaro Linux x86_64
Host: 82V2 (Slim 7 ProX 14ARH7)
Kernel: Linux 6.9.9-1-MANJARO
Display (CSO1410): 3072x1920 @ 120Hz (as 1536x960) [Built-in]
DE: KDE Plasma 6.0.5
WM: KWin (Wayland)
CPU: AMD Ryzen 9 6900HS Creator Edition (16) @ 4.93 GHz
GPU 1: AMD Radeon 680M [Integrated]
GPU 2: NVIDIA GeForce RTX 3050 Mobile [Discrete]
Memory: 27.10 GiB

### Printer

any

### How to reproduce

1. Click the 'Edit' button on any printer or filament preset
2. Click on another tab in the preset window (ie try to switch from 'Basic information' to 'Machine G-code' in a printer preset)
3. Crash 

### Actual results

Program freezes for a second, then closes itself

### Expected results

Tab switches

### Project file & Debug log uploads

Here's my full log folder containing several crash logs, mostly if not all from this issue. 
[log.zip](https://github.com/user-attachments/files/16396171/log.zip)

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Noisyfox**
This fixes #6261.

### CODE REVIEW

File: src/slic3r/GUI/Tab.cpp [hunk 1]
```diff
@@ -266,7 +266,7 @@ void Tab::create_preset_tab()
     set_tooltips_text();
 
     add_scaled_button(m_top_panel, &m_undo_btn,        m_bmp_white_bullet.name());
-    add_scaled_button(m_top_panel, &m_undo_to_sys_btn, m_bmp_white_bullet.name());
+    //add_scaled_button(m_top_panel, &m_undo_to_sys_btn, m_bmp_white_bullet.name());
```
Comment (SoftFever) [inline src/slic3r/GUI/Tab.cpp:269]: @Noisyfox what is this change for? I don't quit follow the changes in this file.

File: src/slic3r/GUI/Field.hpp [hunk 2]
```diff
@@ -506,6 +506,7 @@ class PointCtrl : public Field {
 	TextInput*      	x_input{nullptr};
 	TextInput*      	y_input{nullptr};
 
+    wxWindow*       window{nullptr};
```
Comment (SoftFever) [inline src/slic3r/GUI/Field.hpp:509]: Nice one

File: src/slic3r/GUI/Tab.cpp (hunk 1 above)
Comment (Noisyfox) [reply src/slic3r/GUI/Tab.cpp:269]: https://github.com/SoftFever/OrcaSlicer/issues/6261#issuecomment-2284606337
For fixing this. This button is never used, and somehow on Linux it's shown, but not on Windows. So I think it's better to remove it completely.

**Review by SoftFever — APPROVED** [review]
Looks good
Thank you

**Pull request merged**